under_development/tensorflow_iris_byom/tensorflow_BYOM_iris.ipynb

```diff
@@ -341,34 +310,32 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
```

Comment:
In the above:
- Replace "instance for this training" with "instance for this example."  Just to avoid confusion with the model training process.
- Can we consistently expect 1?  I got 2 with a score of 0.5236916542053223.  Didn't have time to dig in but I expect it's due to initial randomization.
- Instead of "Run a sample prediction on a sample..." can we change to "Run and example prediction on a sample..." just to avoid confusion?
- Change "continied use of" to "to continue using".